packages/manager/modules/vps/src/dashboard/vps-dashboard.html

```diff
@@ -1,3 +1,15 @@
+<oui-message data-ng-if="$ctrl.isMigrationRequired" data-type="warning">
+    <p
+        data-translate="vps_dashboard_migration_warning"
+        data-translate-values="{date: $ctrl.migrationDate}"
+    ></p>
+    <button
```

Comment:
Looks like it could be a link

Reply:
Yes... This could have been a `<a>` tag with a `href`. But, there is considerable logic being executed and there is also an array that is passed as a parameter. Considering this, a function is being used (instead of a link). I think in this case, a button is a better option, especially because we are executing a function call, than a link. Please let us know if you think otherwise :)